server/methods/core/orders.js

```diff
@@ -62,6 +62,56 @@ export function ordersInventoryAdjust(orderId) {
  */
 export const methods = {
   /**
+   * orders/shipmentPicked
```

Comment:
Not specific to this PR but this file is already >1000 LOC. I wonder if we should start breaking these down?